migrations/etl/2_Mine_manager.sql

```diff
@@ -23,6 +25,168 @@ BEGIN
+            first_name                        ,
+            surname                           ,
+            CASE
+                WHEN phone_no>=10^10 AND phone_no<2*10^10 THEN (phone_no-10^10 )::varchar--Remove country code for phone_no
+                WHEN phone_no>10^9 AND phone_no<10^10 THEN (phone_no)::varchar
+                ELSE 'Unknown'
+            END AS phone_no ,
+            email           ,
+            effective_date
+        FROM existing_person_info
+    ),
+    --Format phone number field to match MDS party table schema
+    existing_person_info_format_phone AS (
+        SELECT
+            party_guid      ,
+            person_combo_id ,
+            first_name      ,
+            surname         ,
+            CASE
+                WHEN phone_no = 'Unknown' THEN phone_no
+                ELSE
+                    SUBSTRING(phone_no from 1 for 3)||'-'||
+                    SUBSTRING(phone_no from 4 for 3)||'-'||
+                    SUBSTRING(phone_no from 7 for 4)
+            END AS phone_no ,
+            email           ,
+            effective_date
+        FROM existing_person_info_get_phone
+    ),
+    --7. Complete list of person info for existing manager record
+    person_info AS (
+        SELECT * FROM existing_person_info_format_phone
+        UNION
+        SELECT
+            party_guid      ,
+            person_combo_id ,
+            first_name      ,
+            surname         ,
+            phone_no        ,
+            email           ,
+            effective_date
+        FROM ETL_MANAGER
+        WHERE person_combo_id IN (
+            SELECT  DISTINCT person_combo_id
+            FROM    existing_manager
+        )
+    ),
+    --8. Complete list of existing regional mine manager
+    existing_manager_info AS (
+        SELECT
+            manager.mine_no         ,
+            manager.person_combo_id ,
+            manager.mgr_combo_id    ,
+            person.party_guid       ,
+            person.first_name       ,
+            person.surname          ,
+            person.phone_no         ,
+            person.email            ,
+            person.effective_date
+        FROM existing_manager manager
+        INNER JOIN person_info person
+            ON person.person_combo_id = manager.person_combo_id
+    ),
+    --9. Update info for a given party at a specific mine
+    updated_rows AS (
+        UPDATE ETL_MANAGER
+        SET first_name      = mms.first_name     ,
+            surname         = mms.surname        ,
+            phone_no        = mms.phone_no       ,
+            email           = mms.email          ,
+            effective_date  = mms.effective_date ,
+            person_combo_id = mms.person_combo_id,
+            mgr_combo_id    = mms.mgr_combo_id
+        FROM existing_manager_info mms
+        WHERE
+            ETL_MANAGER.mine_no = mms.mine_no
+            AND
+            ETL_MANAGER.party_guid = mms.party_guid
+        RETURNING 1
```

Comment:
Sort of, yeah. It populates the CTE with 1s, giving us an accurate count of how many records were touched via the `SELECT COUNT(*)` query that follows. It allows us to avoid doing another query with the same conditions as the `INSERT` or `UPDATE` to get the count of effected records.